tools/distpartitioning/data_shuffle.py

```diff
@@ -534,12 +543,14 @@ def gen_dist_partitions(rank, world_size, params):
     print('[Rank: ', rank, '] Starting distributed data processing pipeline...')
 
     #init processing
+    schema_map = read_json(params.input_dir+'/'+params.schema)
```

Comment:
shouldn't you use `os.path.join()` to get the full path?

Reply:
Agreed. changed appropriately.